addons/payment_transfer/models/payment.py

```diff
@@ -18,35 +18,6 @@ class TransferPaymentAcquirer(models.Model):
     def transfer_get_form_action_url(self):
         return '/payment/transfer/feedback'
 
-    def _format_transfer_data(self):
```

Comment:
We removed the feature "Display on Footer" on bank accounts because it's easier to directly configure your footer with plain text: 1/ easier to find the option rather than setting bank journals, and 2/ the way to display bank data are very country specific. (so the way Odoo renders it is wrong 50% of the time: IBAN/BIC/Swift, RIB, bank address of not, ...)

The same logic applies to default value of wire transfer (which was the only other feature that was using the "Show on Footer" option) --> It's better that the user sets its value himself directly instead of providing something wrong.

Note: This code was even worse, the value was created after the create(), so the user did not even saw this default message when creating the document.

More over, by removing this, it allows to remove the "Display on Footer" option on bank journals.